docs/source/quickstart.rst

```diff
@@ -25,6 +25,9 @@ You install MLflow by running:
     installing Python 3 through the `Homebrew <https://brew.sh/>`_ package manager using
     ``brew install python``. (In this case, installing MLflow is now ``pip3 install mlflow``).
 
+In order to use some of MLflow features (ML librairies, storage options, ...) you may need to install extra librairies.
```

Comment:
```suggestion
In order to use some of MLflow features (ML librairies, storage options, ...), you may need to install extra libraries.
```

Reply:
Can we provide an example? E.g., "For example, the `mlflow.tensorflow` module requires TensorFlow to be installed."

Reply:
excellent idea :)